narayana-spring-boot-starter-it/src/test/java/dev/snowdrop/boot/narayana/generic/GenericRecoveryIT.java

```diff
@@ -117,7 +117,7 @@ void testCrashBeforeCommit() throws Exception {
         assertEntriesAfterCrash(this.entriesService.getEntries());
 
         await("Wait for the recovery to happen")
-                .atMost(Duration.ofSeconds(60))
+                .atMost(Duration.ofSeconds(30))
```

Comment:
Why this change?

Reply:
Changed back to 30 because a better workaround is now available with Agroal 2.5. look at flushonclose. It's only for a stupid behavior of H2 with shared connections between recovery and normal purposes.